### ISSUE TITLE
/login/oauth/access_token seems to be a GET instead of a POST

### ISSUE THREAD

**Issue by toolness** [mozilla/id.webmaker.org#116]
The [documentation](https://github.com/mozilla/id.webmaker.org/blob/develop/docs/oauth.md#3-exchange-authorization-code-for-access-token) states that `/login/oauth/access_token` should be a POST endpoint with a form-encoded body, but after attempting to do this and getting a 404, I poked at the [source code](https://github.com/mozilla/id.webmaker.org/blob/develop/web/server.js#L117-L131) and noticed that the route currently appears to be a `GET` endpoint that takes querystring arguments instead:

```js
    {
      method: 'GET',
      path: '/login/oauth/access_token',
      config: {
        validate: {
          query: {
            code: Joi.string().required(),
            client_id: Joi.string().required(),
            client_secret: Joi.string().required(),
            grant_type: Joi.any().valid('authorization_code').required()
          },
```

Should I just use this for now, or should the code be changed? Or am I looking at the wrong code?

**Comment by jbuck** [comment]
Wow, we managed to implement that incorrectly. Nice. Yes, the code should be changed to a POST with a form-encoded body.

@cadecairos can you fix this up?

**Comment by toolness** [comment]
Another funky thing I'm noticing is that in the success case, the [code](https://github.com/mozilla/id.webmaker.org/blob/develop/web/server.js#L176) appears to be redirecting to the callback URL instead of providing a JSON blob like the docs state...

**Comment by cadecairos** [comment]
fixed by #122

**Comment by toolness** [comment]
Hooray thanks!
